apps/web/src/server/applications/create/case/applications-create-case.controller.js

```diff
@@ -63,10 +63,13 @@ export async function updateApplicationsCreateCaseName(
 	const { applicationId } = response.locals;
 	const { description, title } = body;
 	const payload = bodyToPayload(body);
-
-	const { errors: apiErrors, id: updatedApplicationId } = await (applicationId
+	const action = applicationId
 		? () => updateApplicationDraft(applicationId, payload)
-		: () => createApplicationDraft(payload, session))();
+		: () => createApplicationDraft(payload, session);
```

Comment:
this was a bug (unrelated with the dashboard page but I was able to find it thanks to it).
in case of FE validation errors the draft was being created anyway